source/azure_iot_hub_client.c

```diff
@@ -583,6 +650,33 @@ void AzureIoTHubClient_Deinit( AzureIoTHubClient_t * pxAzureIoTHubClient )
 }
 /*-----------------------------------------------------------*/
 
+AzureIoTHubClientResult_t AzureIoTHubClient_SetSymmetricKey( AzureIoTHubClient_t * pxAzureIoTHubClient,
+                                                             const uint8_t * pucSymmetricKey,
+                                                             uint32_t ulSymmetricKeyLength,
+                                                             AzureIoTGetHMACFunc_t xHmacFunction )
+{
+    AzureIoTHubClientResult_t xResult;
+
+    if( ( pxAzureIoTHubClient == NULL ) ||
+        ( pucSymmetricKey == NULL ) || ( ulSymmetricKeyLength == 0 ) ||
+        ( xHmacFunction == NULL ) )
+    {
+        AZLogError( ( "AzureIoTHubClient set symmetric key failed: Invalid argument" ) );
```

Comment:
should this just be the function name?
```suggestion
        AZLogError( ( "AzureIoTHubClient_SetSymmetricKey failed: Invalid argument" ) );
```